web/package.json

```diff
@@ -187,7 +187,6 @@
   "lint-staged": {
     "*": "prettier -uw --cache",
     "*.{ts,tsx}": [
-      "eslint --cache --fix",
```

Comment:
This prevented me from pushing the code until all ESlint errors are fixed. Will add it back once they are fixed.